canonical-web-and-design/maas.io
# Manually update the API docs with 2.8

## Done
Manually update the API docs with 2.8

## QA
- Go to /docs/api
- Check the page loads and renders correctly
